content/en/docs/ambient/architecture/index.md

```diff
@@ -9,57 +9,109 @@ owner: istio/wg-networking-maintainers
+
+In order for your `istio.io/use-waypoint` label value to be effective, you have to ensure waypoint is configured for the endpoint which is using it. By default waypoints accept traffic for service endpoints. For example, when you label a pod to use a specific waypoint via the `istio.io/use-waypoint` label, the waypoint should be labeled `istio.io./waypoint-for` with the value `workload` or `all`.
+
+### Layer 7 policy attachment to waypoints
+
+You can attach Layer 7 policies (such as `AuthorizationPolicy`, `RequestAuthentication`, `Telemetry`, `WasmPlugin`, etc) to your waypoint using `targetRef`.
+
+- To attach L7 policy to the entire waypoint, set `Gateway` as the `targetRefs` value. The example below shows how to attach the `viewer` policy
+to the waypoint named `waypoint` for the `default` namespace:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: viewer
+      namespace: default
+    spec:
+      targetRefs:
+      - kind: Gateway
+        group: gateway.networking.k8s.io
+        name: waypoint
+    {{< /text >}}
+
+- To attach L7 policy to a specific service within the waypoint, set `Service` as the `targetRefs` value. The example below shows how to attach
+the `productpage-viewer` policy to the `productpage` service in the `default` namespace:
+
+    {{< text yaml >}}
+    apiVersion: security.istio.io/v1beta1
+    kind: AuthorizationPolicy
+    metadata:
+      name: productpage-viewer
+      namespace: default
+    spec:
+      targetRefs:
+      - kind: Service
+        group: ""
+        name: productpage
+    {{< /text >}}
+
 ## Traffic routing
 
 In {{< gloss "ambient" >}}ambient mode{{< /gloss >}}, workloads can fall into 3 categories:
-1. **Uncaptured:** this is a standard pod without any mesh features enabled.
-1. **Captured:** this is a pod that has traffic intercepted by {{< gloss >}}ztunnel{{< /gloss >}}. Pods can be captured by setting the `istio.io/dataplane-mode=ambient` label on a namespace.
-1. **Waypoint enabled:** this is a pod that is "Captured" *and* has a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} deployed.
-  A waypoint will, by default, apply to all pods in the same namespace.
-  It can optionally be set to apply to only a specific service account with the `istio.io/for-service-account` annotation on the `Gateway`.
-  If there is both a namespace waypoint and service account waypoint, the service account waypoint takes precedence.
+1. **Out of Mesh:** this is a standard pod without any mesh features enabled.
+1. **In Mesh:** this is a pod that has traffic intercepted at the Layer 4 level by {{< gloss >}}ztunnel{{< /gloss >}}. In this mode, L4 policy can be enforced for pod traffic. This mode can be enabled for a pod by setting the `istio.io/dataplane-mode=ambient` label on the pod's namespace. This will enable *in mesh* mode for all pods in that namespace.
```

Comment:
```suggestion
1. **In Mesh:** this is a pod that has traffic intercepted at the Layer 4 level by {{< gloss >}}ztunnel{{< /gloss >}}. In this mode, L4 policies can be enforced for pod traffic. This mode can be enabled for a pod by setting the `istio.io/dataplane-mode=ambient` label on the pod's namespace. This will enable *in mesh* mode for all pods in that namespace.
```